template.js

```diff
@@ -424,12 +424,10 @@ function addEcommerceData(eventData, mappedData) {
 }
 
 function addUserData(eventData, mappedData) {
-  const address =
-    eventData.user_data &&
-    eventData.user_data.address &&
-    eventData.user_data.address[0]
-      ? eventData.user_data.address[0]
-      : {};
+  let address = {};
+  if(eventData.user_data && eventData.user_data.address) {
+    address = eventData.user_data.address[0] || eventData.user_data.address
```

Comment:
Add address type object check

Reply:
add if for 
eventData.last_name, 
eventData.first_name, 
eventData.region, 
eventData.postal_code, 
eventData.country 
in case data will come without user_data object

+ add if  for 
eventData.user_data.last_name, 
eventData.user_data.first_name, 
eventData.user_data.region, 
eventData.user_data.postal_code, 
eventData.user_data.country 
in case data will come without address object